geopandas/geodataframe.py

```diff
@@ -575,6 +576,25 @@ def merge_geometries(block):
 
         return aggregated
 
+    # overrides GeoPandasBase method
+    def explode(self):
+        """
+        Explode multi-part geometries into multiple single geometries.
+
+        Single rows can become multiple rows.
+        This is analogous to PostGIS's ST_Dump(). 
+
+        Returns
+        ------
+        A GeoDataFrame with rows containing MultiPoints expanded into single
+        Point rows. The rest of the original columns are preserved.
```

Comment:
I would make this a bit more generic, as this also works for eg MultiPolygons (so not only for MultiPoints) 
Also, I would maybe put this explanation a few lines above (I would expand the "Single rows can become multiple rows" explanation)